homeassistant/components/binary_sensor/octoprint.py

```diff
@@ -7,45 +7,34 @@
 import logging
 
 import requests
-import voluptuous as vol
 
-from homeassistant.const import CONF_NAME, CONF_MONITORED_CONDITIONS
-from homeassistant.components.binary_sensor import (
-    BinarySensorDevice, PLATFORM_SCHEMA)
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.octoprint import BINARY_SENSOR_TYPES
+from homeassistant.components.binary_sensor import BinarySensorDevice
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['octoprint']
 DOMAIN = "octoprint"
```

Comment:
This platform belongs to the binary_sensor domain. Remove this and import the octoprint domain as another name.